es/README.md

```diff
@@ -1,36 +1,70 @@
-## Install, configure and start Elasticsearch
+# Install, configure and start Elasticsearch
 
-### Manual installation
+## Manual installation
 
-Download Elasticsearch (at least 7.6.2 for Geonetwork 3.8.x) from https://www.elastic.co/fr/downloads/elasticsearch
-and copy it to the ES module. eg. es/elasticsearch-7.6.2
+1. Download Elasticsearch (at least 7.6.2 for Geonetwork 4.0.x) from https://www.elastic.co/downloads/elasticsearch
+and copy to the ES module. eg. es/elasticsearch-7.6.2
  
-Start ES using:
+2. Start ES using:
 
-```shell script
-./bin/elasticsearch
-```
+   ```shell script
+   ./bin/elasticsearch
+   ```
 
+3. Check that elasticsearch is running by visiting http://localhost:9200 in a browser
 
-### Maven installation
+## Maven installation
 
-Maven can take care of the installation steps:
-* download
-* initialize collection
-* start
+1. Maven can take care of the installation steps:
 
-Use the following commands:
+   * download
+   * initialize collection
+   * start
 
-```shell script
-cd es
-mvn install -Pes-download
-mvn exec:exec -Des-start
-```
+2. Use the following commands:
+
+   ```shell script
+   cd es
+   mvn install -Pes-download
+   mvn exec:exec -Des-start
+   ```
+3. Check that elasticsearch is running by visiting http://localhost:9200 in a browser
+
+## Docker installation
+
+1. Use docker pull to download the image (you can check version in the :file:`pom.xml` file):
+
+   ```
+   docker pull docker.elastic.co/elasticsearch/elasticsearch:7.6.2
+   ```
+
+2. Use docker run, leacing 9200 available:
```

Comment:
What is `leacing` ?

Reply:
I am going to assume "leaving port 9200 available"